portfolio/src/main/webapp/index.html

```diff
@@ -6,7 +6,7 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body style="background-color: #f9f9f0">
```

Comment:
This should go in CSS.

https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns